webapp/packages/core-connections/src/ConnectionAuthService.ts

```diff
@@ -12,6 +12,8 @@ import { CommonDialogService } from '@cloudbeaver/core-dialogs';
 import { Connection, ConnectionInfoResource } from './ConnectionInfoResource';
 import { DatabaseAuthDialog } from './DatabaseAuthDialog/DatabaseAuthDialog';
 
+const SSH_TUNNEL_ID = 'ssh_tunnel';
```

Comment:
import from `Administration/Connections/ConnectionForm/SSH/SSH_TUNNEL_ID.ts` 
maybe better to move this const to `NetworkHandlerResource`